src/net/util/show_packet.h

```diff
@@ -20,7 +20,7 @@ static inline void show_packet(uint8_t *raw, int size, char *title) {
 	if (!&mod_logger)
 		return;
 
-	if (&mod_logger.logging &&
+	if (//&mod_logger.logging &&
```

Comment:
I'd suppose this and attr_len fix should be removed from this PR, they will be merged as #1024